PWGJE/EMCALJetTasks/Tracks/AliAnalysisTaskEmcalJetEnergySpectrum.cxx

```diff
@@ -613,15 +613,19 @@ Double_t AliAnalysisTaskEmcalJetEnergySpectrum::GetDeltaPtRandomCone()
 
     } while (dLJ < 0.45 || dSLJ < 0.45);
 
-    AliVTrack *tmpTrack = 0x0;
+    AliAODTrack *tmpTrack = 0x0;
 
     for (Int_t i = 0; i < partcont->GetNAcceptedParticles(); i++)
     {
 
         if (!partcont->GetParticle(i))
             continue;
         
-        tmpTrack = static_cast<AliVTrack *>(partcont->GetParticle(i));
+        //tmpTrack = static_cast<AliVTrack *>(partcont->GetParticle(i));
+        tmpTrack = (AliAODTrack *)partcont->GetParticle(i);
+
+        if (!((tmpTrack)->TestFilterBit(1 << 8)) && !((tmpTrack)->TestFilterBit(1 << 9)))
+            continue;
```

Comment:
Please use hybrid track selector functions in AliAODTrack (IsHybrid...) instead of direct cuts on the filter bit.

Reply:
Done